drivers/oled/ssd1306_sh1106.c

```diff
@@ -301,7 +301,8 @@ void oled_render(void) {
     oled_on();
 
     uint8_t update_start = 0;
-    while (oled_dirty) { // render all dirty blocks
+    uint8_t dirty_count  = 0;
+    while (oled_dirty && dirty_count++ < OLED_UPDATE_PROCESS_LIMIT) { // render all dirty blocks (up to the configured limit)
```

Comment:
Is `oled_dirty` still needed with this new variable? I would think the same could be expressed by `dirty_count` being nonzero.

Reply:
We still need to track 2 things,
- which blocks are dirty
- how many have been processed in a given call to `oled_render`

Ive renamed the variable to maybe convey that slightly better.

Reply:
Ah I see, I thought `oled_dirty` was a bool.